src/input/Binding.zig

```diff
@@ -341,7 +341,8 @@ pub const Action = union(enum) {
     /// the direction given. For example `new_split:up`. Valid values are left, right, up, down and auto.
     new_split: SplitDirection,
 
-    /// Focus on a split in a given direction. For example `goto_split:up`. Valid values are left, right, up, down, previous and next.
```

Comment:
This is the only docs line that crosses the 120 char boundary. I know this is probably not a hard requirement but I thought it will make this more readable in the IDE.